doc/src/tutorial/intro.rst

```diff
@@ -163,21 +161,17 @@ Find the eigenvalues of `\left[\begin{smallmatrix}1 & 2\\2 &
 2\end{smallmatrix}\right]`.
 
  >>> Matrix([[1, 2], [2, 2]]).eigenvals()
- ⎧      ____         ____       ⎫
- ⎪3   ╲╱ 17        ╲╱ 17    3   ⎪
- ⎨─ + ──────: 1, - ────── + ─: 1⎬
- ⎪2     2            2      2   ⎪
- ⎩                              ⎭
+ ⎧3   √17       √17   3   ⎫
```

Comment:
This one doesn't look so bad though, compared to the three digit one we discussed in the pr comments. Hmm. I guess if SymPy always puts a multiplication dot between values then maybe the non-overline would work.